include/tlapack/lapack/gelqf.hpp

```diff
@@ -26,14 +26,14 @@ struct gelqf_opts_t : public workspace_opts_t<> {
     inline constexpr gelqf_opts_t(const workspace_opts_t<>& opts = {})
         : workspace_opts_t<>(opts){};
 
-    idx_t nb = 32;  ///< Block size
```

Comment:
Not a big deal here, but I want to mention that `///<` is the way Doxygen recognizes comments for attributes of classes.